addons/binding/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/KebaBindingConstants.java

```diff
@@ -25,7 +25,7 @@
     public static final String BINDING_ID = "keba";
 
     // List of all Thing Type UIDs
-    public static final ThingTypeUID THING_TYPE_KECONTACTP20 = new ThingTypeUID(BINDING_ID, "kecontactp20");
+    public static final ThingTypeUID THING_TYPE_KECONTACTP20 = new ThingTypeUID(BINDING_ID, "kecontact");
```

Comment:
Doesn't the README needs to be updated, too?
Is this a breaking change that should be mentioned in an upcoming 2.3 release notes?

Reply:
Ooops. forgot that one, commit is here below

Reply:
The note for the release notes should be added [here](https://github.com/openhab/openhab-distro/wiki/%5BDRAFT%5D-openHAB-2.3-Release-Notes#addons) (in the section "The following changes were done on existing 2.x add-ons").

Reply:
Please do NOT change the ThingTypeUID - or support both.
This will break all installations already out there. Keep it compatible!!!!

Reply:
@ThomasBrodkorb Since the P20 and P30 are very similar in terms of hardware and features, and since there are quite a few different firmware revisions installed, it is not practical to define a ThingType for every {hardware, firmware}-tuple, and therefore everything is unified in a single ThingType with differentiation of features based on the hardware type and firmware versions returned by the Keba charger station.

Reply:
Maybe I got misunderstood: I did not ask for having a thingtype for each hardware/firmware version.
What I asked for was to keep the current kecontactp20 as an alias (if such exists).

In the Openhabian environment, you are not under control of the updates (update all or nothing).
So any change in the naming of the binding will cause end user systems to fail.

Reply:
So I understand correctly this change is already out there and should not be included anymore in the 2.3 updates. Although it would be great if more time could be spent on changing it such that it works also with the old definition.